README.md

```diff
@@ -1 +1,207 @@
+
+##### Guidelines for good commit messages
+1. Keep it **short** (less than 150 characters total)
+    - Committing **fewer changes** at a time can help with this
+2. Use the **imperative** mood
+    - This convention aligns with commit messages generated by commands like git merge and git revert
+    - Consistency enhances speed of reading comprehension
+    - Tends to be more concise than the other moods
+3. Specify the **type of commit** (Link to [ISSUE](#issues)):
+    - feat: The new feature you're adding to a particular application
+    - fix: A bug fix
+    - style: Feature and updates related to styling
+    - refactor: Refactoring a specific section of the codebase
+    - test: Everything related to testing
+    - docs: Everything related to documentation
+    - chore: Regular code maintenance.[ You can also use emojis to represent commit types]
+4. Remove unnecessary punctuation marks
+5. Do not end the subject line with a period
+6. Capitalize the subject line and each paragraph
+7. Use the body to explain what changes you have made and why you made them.
+8. Information in commit messages:
+    - **Issue it fixes**
+    - **Why** the change was made
+    - **Where** the change was made
+    - If it's a fix, describe the bug
+9.  Be **CONCISE**.
+
+#### Pull Requests
+
+> "A pull request is a proposal to merge a set of changes from one branch into another. In a pull request, collaborators can review and discuss the proposed set of changes before they integrate the changes into the main codebase. Pull requests display the differences, or diffs, between the content in the source branch and the content in the target branch." [About Pull Requests in GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+
+Once you've made commits and are done working on the feature your branch was for, you should make a pull request. A pull request is meant to allow your teammates to review your changes, give you feedback, and the merge all the functional code into one central location (main branch).
+
+## README, Markdown, and Report Submission
+
+For this project, you're expected to submit a report that is similar to those you've done for projects 1 and 2, with the proposal submitted in 3a included. To centralize this with your source code and video, you will write your Report in Markdown as a README file. Since you're supposed to create your project repository from this template, edit the existing README.md file, remove all the contents that are currently there, and replace them with your report.
+
+Putting the README.md in your root or docs directory will automatically be recognized by GitHub and will be shown to anyone who visits the repository. For all documents, including README files, commit messages and issues, GitHub uses Markdown for formatting.
+
+### Important Elements
```

Comment:
All of the markdown stuff should be its own readme file.
I think we can have that "resources folder" and within that you have markdown file as well as images folder

Reply:
Made separate directory for Markdown info (linked it in the README) and put the template repo in there, too.